pkgs/os-specific/linux/kernel/generate-config.pl

```diff
@@ -13,18 +13,18 @@
 use IPC::Open2;
 use Cwd;
 
-my $wd = getcwd;
-
+# exported via nix
 my $debug = $ENV{'DEBUG'};
 my $autoModules = $ENV{'AUTO_MODULES'};
 my $preferBuiltin = $ENV{'PREFER_BUILTIN'};
-    
+my $ignoreConfigErrors = $ENV{'ignoreConfigErrors'};
+my $buildFolder=$ENV{'BUILD_FOLDER'};
```

Comment:
Missing spaces around ` = `

Reply:
I'd prefer this to be called `buildRoot` since it's called that in `manual-config.nix` as well.

Reply:
fixed